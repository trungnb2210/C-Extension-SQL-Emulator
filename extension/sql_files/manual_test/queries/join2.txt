SELECT item, size, count, price, quantity
FROM things JOIN prices ON object == item
