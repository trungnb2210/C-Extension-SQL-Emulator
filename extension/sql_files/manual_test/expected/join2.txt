QUERY RESULT:
  item    size   count   price   quantity 
-------------------------------------------
| Box   | 100  | 2     | 100   | 2        | 
| Box   | 100  | 2     | 250   | 7        | 
| Tire  | 120  | 5     | 120   | 1        | 
| Table | 300  | 1     | 300   | 5        | 
-------------------------------------------
